package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@blackbaud/skyux-lib-restricted-view",
+  "version": "1.0.0-rc.1",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=6.11.1"
+  },
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test && skyux e2e && skyux build && npm run build",
+    "build": "skyux build-public-library",
+    "start": "skyux serve"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/animations": ">=7.2.2",
+    "@angular/common": ">=7.2.2",
```

Comment:
I would also avoid using `>=` with peer dependencies; you probably only want to support minor versions `^`.